bin/plugin/lib/milestone.js

```diff
@@ -59,7 +58,7 @@ async function getIssuesByMilestone(
 	state,
 	closedSince
 ) {
-	const options = octokit.issues.listForRepo.endpoint.merge( {
+	const responses = octokit.paginate.iterator( octokit.issues.listForRepo, {
 		owner,
 		repo,
 		milestone,
```

Comment:
Seems like TS is complaining since it expects a `string | undefined` here -- but [the docs](https://octokit.github.io/rest.js/v18#issues-list-for-repo) say that a number should be okay :thinking:

Reply:
This would indeed fix the issue, but I think we might want to report this upstream instead:

```diff
diff --git a/bin/plugin/lib/milestone.js b/bin/plugin/lib/milestone.js
index 226d968980..7fe20cfeee 100644
--- a/bin/plugin/lib/milestone.js
+++ b/bin/plugin/lib/milestone.js
@@ -43,7 +43,7 @@ async function getMilestoneByTitle( octokit, owner, repo, title ) {
  * @param {GitHub}     octokit       Initialized Octokit REST client.
  * @param {string}     owner         Repository owner.
  * @param {string}     repo          Repository name.
- * @param {number}     milestone     Milestone ID.
+ * @param {string}     milestone     Milestone ID.
  * @param {IssueState} [state]       Optional issue state.
  * @param {string}     [closedSince] Optional timestamp.
  *
```

Reply:
> I think we might want to report this upstream instead

This is indeed a bug in GitHub's OpenAPI spec, from which all the @octokit types and method names are generated:

```yml
      - name: milestone
        description: If an `integer` is passed, it should refer to a milestone by
          its `number` field. If the string `*` is passed, issues with any milestone
          are accepted. If the string `none` is passed, issues without milestones
          are returned.
        in: query
        required: false
        schema:
          type: string
```

I filed an issue here: https://github.com/github/rest-api-description/issues/226

Reply:
Wow, beat me to it! Thanks a lot @gr2m!